src/commands/admin/config/appbutton/create.ts

```diff
@@ -100,7 +100,7 @@ export default class ConfigAppbuttonCreateCommand extends BotCommand {
 							apps.map(a => ({
 								label: a.name,
 								value: a.id.toString(),
-								description: a.description ?? undefined
+								description: (a.description.length <= 100 ? a.description : a.description.slice(0,97)+"...") ?? undefined
```

Comment:
This wouldn't work because app.description can be null
It would probably have to be something like
```ts
(app.description?.length > 100 ? <trim it> : app.description) ?? undefined